src/main/java/org/sagebionetworks/web/client/presenter/SearchUtil.java

```diff
@@ -19,7 +22,7 @@
 	 * @param place
 	 * @return
 	 */
-	public static Synapse willRedirect(Search place) {
+	public static Place willRedirect(Search place) {

```

Comment:
Based on the search term, might return a Synapse place (if the query term starts with 'syn') or a PeopleSearch place (if it starts with '@')